mobility_pipeline/lib/overlap.py

```diff
@@ -0,0 +1,9 @@
+from shapely.geometry import Polygon
+
+def computeOverlap(enclosing, polygons):
+    overlapPercentages = []
```

Comment:
Can you add a docstring here as well? It should describe what this function does, versus the earlier docstring that describes what the whole file does.